### DISCUSSION TITLE
More filters and sort options

### DISCUSSION THREAD

**Discussion by janstehno** [janstehno/cotw-companion#6] [Ideas]
This will affect all the lists probably. Mainly for Catch book and Trophy lodge lists (suggested was sort option for trophy rating). Another could be options for weapons (filter rifles only for example), or for animals (filter class 6 animals only for example).

**Comment by janstehno** [comment]
Added in 1.7.3. Suggestions for more options are more than welcome.

**Reply by Etney-Yente** [reply]
Hi, is there an option to add a “reserves” filter in the wildlife section of the app? This way we could toggle Medved for example and have a high level overview of the animals for that reserve (class, max level and diamond score). Personally I can’t remember every diamond score there is so I mainly have to use quick search for now. Thanks

**Comment by N3-Oti** [comment]
Hi developers,

First off, thank you for creating and maintaining this very useful cotw-companion tool!

I have a couple of suggestions regarding the filtering features that I believe could enhance its usability, especially for completing challenges:

Tags/Filters for Challenge Categories:

Weekly and Daily Challenges often require hunting animals based on specific categories, such as "Horned" or "Goats", among others.

It would be extremely helpful if we could filter animals using tags corresponding to common challenge criteria like these. This would make it much easier to identify target animals for various challenges.

AND Search Capability:

Currently, the filtering seems to operate primarily on an OR basis when multiple criteria are selected.

Adding an AND search/filter logic would be very beneficial. This would allow for more precise queries.

Use Case 1 (Weapons): Filtering weapons suitable for a specific range of classes, for example, classes 4 through 7 (not just 4 OR 7).

Use Case 2 (Planning Hunts): Narrowing down potential animals for a challenge AND filtering by specific maps to efficiently plan where to hunt them.

Implementing these features would make planning for challenges and general gameplay significantly easier.

Thank you for considering these suggestions!

**Reply by janstehno** [reply]
Hello, I am glad the app suits your needs. Thank you for the suggestions. I am definitely up for more filters and options to find what you need as quickly as possible. And I really like your ideas. They will be most likely implemented within some of the QoL updates I am working on right now.

**Comment by M1GHTY-M4VS** [comment]
Filter for Animal Types could be biological systematics, one of the Challenge could be All Bunny Types for Easter, the modern taxonomic Ranking via Carl von Linne for example has lifeforms of Animalia via a Reach (Regnum), and puts them Into a certain hierarchy, the Animal Class (Classis), Order (Ordo), Stem (Genus), before the subtype each Animal makes it it's Type (species) own in Game.

You could Group for example Alligators and Crocodiles as Crocodylia but also as Reptile as upper category with distant relatives of Birds (Aves) of a higher relation distant Evolution.

Modern Classical Hunters also group the Animal by Domain like the big five of Africa, 29 Animals of Savanna, American 7 Big Sheep, World Stag List (Deer Species of the World) with sub division by Continental or Country or Linguistic Commonalities (i.e Anglosphere).
But could also make a destinction to Climate.

For Weapon Ammo it could have a Origin i. e mm are often European Countries, while the US has both Odd and Fractional Calibers, the Mosin and SKS were Soviet so there could be a case for Warsaw Pact and Nato or Designed for hunting or PRC. For Weapons having Categories for Bolt Action, Break Action, Single Shot, Magazines, Rapid Fire could also be useful for some that rotate and favour something. 

Animal Behaviour could be another thing like classify Tigers as active (against theHunter) Predator. Red Deer as Herders but with trample Damage on Fleeing), so you could highlight a danger implication to the Maps Subregions.

The Maps Feature on the App throws me off sometimes with the Colour Coding a bit when Red Deer and Red Fox have no Redish Text or if a dangerous Animal has a Green Text, since the Colours are linear by Alphabet I think instead of by Size or Difficulty
